fs/remote/resolver.go

```diff
@@ -120,13 +182,23 @@ func newFetcher(ctx context.Context, hosts source.RegistryHosts, refspec referen
 	for _, host := range reghosts {
 		if host.Host == "" || strings.Contains(host.Host, "/") {
 			rErr = errors.Wrapf(rErr, "invalid destination (host %q, ref:%q, digest:%q)",
-				host.Host, refspec, digest)
+				host.Host, fc.refspec, digest)
 			continue // Try another
 
 		}
 
 		// Prepare transport with authorization functionality
 		tr := host.Client.Transport
+
+		if rt, ok := tr.(*rhttp.RoundTripper); ok {
+			rt.Client.RetryMax = fc.maxRetries
+			rt.Client.RetryWaitMin = fc.minWaitMSec
+			rt.Client.RetryWaitMax = fc.maxWaitMSec
+			rt.Client.Backoff = backoffStrategy
+			rt.Client.CheckRetry = retryStrategy
+			rt.Client.Logger = nil // disable logging every request
```

Comment:
Logger should be disabled when the RoundTripper is created (i.e. `rhttp.NewClient()`).

Reply:
Done. Thanks.